doc/Telco-PaaS-definition/TelcoPaaSDefChart.md

```diff
@@ -1,10 +1,10 @@
 # XGVela Telco PaaS Definition
 |#|Functions|Description|Software (project name & version)|Telco enhancement point|Tag|Priority|Committer|
 |---|---|---|---|---|---|---|---|
-|1|Container||||Infrastructure||
-|2|Orchestration||||Infrastructure||
-|3|Storage||||Infrastructure||
-|4|Network||||Infrastructure||
+|1|Container|(1) Runtime: support Open Container Initiative (OCI) runtime spec and Kubernetes Container Runtime Interface (CRI) to integrate with K8s for container management; (2) Resource management: memory, CPU, NUMA, HW accelerators; (3) VNF/CNF: support managing VNF and CNF in same cluster|(1) ICN: https://wiki.akraino.org/display/AK/ICN+R3++Release (2) Kubevirt: https://github.com/kubevirt/kubevirt||Infrastructure||
+|2|Orchestration|(1) Orchestration application with K8s native support (2) Orchestration application with K8s Application package manager (e.g. helm) (3)  Cross-sites orchestration|(1) helm: https://github.com/helm/helm (2)EMCO: https://git.onap.org/multicloud/k8s||Infrastructure||
+|3|Storage|(1) Local storage on HDD (Hard Disk Drive) and SSD (Solid State Drive) (2) Remote storage (e.g. iSCSI, NFS etc.) (3) Catalogue/block storage (4) replication and encryption|||Infrastructure||
+|4|Network|(1) Attaching multiple network interfaces to pod (e.g. CNI meta plugin/CNI multiplexer); (2) Managing and attaching pod to multiple virtual networks; (3) Attaching pod to provider network; (4) Service Function Chain (SFC); (5) Management of CNF network; (6) Multiple tenancy; (7) IPv4 and IPv6|(1) OVN-for-K8s NFV controller: https://github.com/opnfv/ovn4nfv-k8s-plugin (2) Multus: https://github.com/intel/multus-cni||Infrastructure||
```

Comment:
maybe you also would like to add [DANM](https://github.com/nokia/danm) as a reference as it supports the management of the CNI networks over an API and provides a cluster wide IPAM.